config/application.yml

```diff
@@ -26,6 +26,8 @@ hedera:
       eth_private_key:
     base_gas_usage: 130000
     gas_per_validator: 54000
+    threshold_account:
```

Comment:
q: threshold_account is the account that we are watching in the transfer watcher right?